skxray/tests/test_diff_roi_choice.py

```diff
@@ -0,0 +1,338 @@
+                     72, 73, 74, 75, 76, 82, 83, 84, 85, 86, 87, 88, 89,
+                     90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 107,
+                     108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118,
+                     119, 120, 121, 122, 123, 124, 125, 126, 132, 133, 134,
+                     135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145,
+                     146, 147, 148, 149, 150, 151, 157, 158, 159, 160, 161,
+                     162, 163, 164, 165, 166, 167, 168, 169, 170, 171, 172,
+                     173, 174, 175, 176, 177, 178, 179, 180, 181, 182, 183,
+                     184, 185, 186, 187, 188, 189, 190, 191, 192, 193, 194,
+                     195, 196, 197, 198, 199, 200, 201, 202, 203, 204, 205,
+                     206, 207, 208, 209, 210, 211, 212, 213, 214, 215, 216,
+                     217, 218, 219, 220, 221, 222, 223, 224, 225, 226, 227,
+                     228, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238,
+                     239, 240, 241, 242, 243, 244, 245, 246, 247, 248, 249,
+                     250, 251, 252, 253, 254, 255, 256, 257, 258, 259, 260,
+                     261, 262, 263, 264, 265, 266, 267, 268, 269, 270, 271,
+                     272, 273, 274, 275, 276, 277, 278, 279, 280, 281, 282,
+                     283, 284, 285, 286, 287, 288, 289, 290, 291, 292, 293,
+                     294, 295, 296, 297, 298, 299, 300, 301, 302, 303, 304,
+                     305, 306, 307, 308, 309, 310, 311, 312, 313, 314, 315,
+                     316, 317, 318, 319, 320, 321, 322, 323, 324, 325, 326,
+                     327, 328, 329, 330, 331, 332, 333, 334, 335, 336, 337,
+                     338, 339, 340, 341, 342, 343, 344, 345, 346, 347, 348,
+                     349, 350, 351, 352, 353, 354, 355, 356, 357, 358, 359,
+                     360, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370,
+                     371, 372, 373, 374, 375, 376, 377, 378, 379, 380, 381,
+                     382, 383, 384, 385, 386, 387, 388, 389, 390, 391, 392,
+                     393, 394, 395, 396, 397, 398, 399, 400, 401, 402, 403,
+                     404, 405, 406, 407, 408, 409, 410, 411, 412, 413, 414,
+                     415, 416, 417, 418, 419, 420, 421, 422, 423, 425, 426,
+                     427, 428, 429, 430, 431, 432, 433, 434, 435, 436, 437,
+                     438, 439, 440, 441, 442, 443, 444, 445, 446, 447, 450,
+                     451, 452, 453, 454, 455, 456, 457, 458, 459, 460, 461,
+                     462, 463, 464, 465, 466, 467, 468, 469, 470, 471, 472,
+                     475, 476, 477, 478, 479, 480, 481, 482, 483, 484, 485,
+                     486, 487, 488, 489, 490, 491, 492, 493, 494, 495, 496]),
+
+    assert_array_almost_equal(q_ring_val_m, q_ring_val)
+    assert_array_equal(num_pixels, num_pixels_m)
+    assert_array_equal(q_inds, np.ravel(q_inds_m))
+    assert_array_equal(pixel_list, pixel_list_m)
```

Comment:
using @tacaswell code to show the rings

fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(12, 6), tight_layout=True)
test_demo(ax1, calibrated_center, q_inds, q_ring_val, pixel_list, img_dim)
  #test_demo(ax2, calibrated_center, q_inds, q_ring_val, pixel_list, img_dim)
plt.show()

![with_q_inds](https://cloud.githubusercontent.com/assets/5701652/6142701/74b6d074-b189-11e4-8baa-95851cbaf805.png)